pkg/network/ovn_kubernetes.go

```diff
@@ -415,6 +415,23 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		return nil, progressing, errors.Wrapf(err, "failed to set IP family %s annotation on daemonsets or statefulsets", ipFamilyMode)
 	}
 
+	// logic to pretty print the clusterNetwork CIDR (possibly only one) in its annotation
+	var clusterNetworkCidrs []string
+	var cidrs string
+	for _, c := range conf.ClusterNetwork {
+		clusterNetworkCidrs = append(clusterNetworkCidrs, c.CIDR)
+	}
+	if len(conf.ClusterNetwork) == 1 {
+		cidrs = clusterNetworkCidrs[0]
+	} else {
+		cidrs = strings.Join(clusterNetworkCidrs, ", ")
+	}
+
```

Comment:
there is no need to check the length, it should be enough to call:
`err = setOVNObjectAnnotation(objs, names.ClusterNetworkCidr, strings.Join(clusterNetworkCidrs, ",")`
Additionally, I think we should ditch the space in `, `

Reply:
seems like overkill, but in the case with just one CIDR, it prints with an ugly trailing `,`

I did get rid of the space after the comma

Reply:
`strings.Join` would not print a sperator if the array is one element:
https://go.dev/play/p/waewUDXbb5O?v=goprev